main.py

```diff
@@ -348,9 +348,9 @@ def parse_human_readable_datetime(cls, v: str) -> datetime:
             raise ValueError('Timezone is required for the cut-off')
         return parsed_cutoff
 
-    @validator('org_name', pre=True)
-    def validate_org_name(cls, v: str, values: dict) -> str | None:
-        if values['account_type'] == AccountType.ORG and not v:
+    @field_validator('org_name', mode='before')
+    def validate_org_name(cls, v: str, values: Any) -> str | None:
```

Comment:
Is there not a more specific type signature we can use here?

Reply:
Is `ValidationInfo` correct?

Reply:
Yeah